### ISSUE TITLE
Unable to build afxgroup clib2 container

### ISSUE THREAD

**Issue by Guddler** [walkero-gr/AmigaGCConDocker#3]
I might well be doing something wrong but I've attached a log of what happens when I try to run make with the following options: 
`make GCC=11 CLIB2_REPO=afxgroup build`

In the resulting container gcc will not run with the following error, which I don't really know is related to the build issues or not:
`qemu-x86_64: Could not open '/lib64/ld-linux-x86-64.so.2': No such file or directory`

[clib2.log](https://github.com/walkero-gr/AmigaGCConDocker/files/12240915/clib2.log)

**Comment by Guddler** [comment]
Established this is an architecture issue. I'm building on an M1 Mac (arm64) and the afxgroup repo only contains amd64 packages. I need to either build from source or switch to a non-native docker build.

**Comment by walkero-gr** [comment]
@Guddler 
I believe that if you use docker on Mac and Arm it can use amd64 images using the qemu, which makes them a little bit slower.
If you need an image that includes GCC 11 and afxgroup's clib2 then try the one I use for the webkit port https://hub.docker.com/r/walkero/webkitondocker

**Comment by Guddler** [comment]
I've been using webkitondocker and the performance really suffers when you're not using the native architecture of the host so ideally I need to build the image instead of using an x86_64 image. From what I can see webkitondocker just uses this one as a base so we're back to the fact this one pulls from an afxgroup repo that doesn't have aarch64.

I'm not sure this really warrants an open issue here. The only "issue" is that your images aren't multi-arch. I'll leave you to re-open if you wish.

**Comment by walkero-gr** [comment]
@Guddler 
I am investigating to make it work on Arm systems. Will let you know if that is possible or not. I already found a fix.
You can open a new ticket asking for the Arm version where we can continue our discussion

**Comment by walkero-gr** [comment]
@Guddler Today I managed to finish the work with the Arm docker images for this repository.
Next, I will work on the WebkitOnDocker image.

**Comment by Guddler** [comment]
Cool - happy to do some testing if you need it, or just provide feedback.

My efforts ground to a halt at the moment. Whichever way I turn I seem to be stuck. Build everything with afxgroup clib2 and I run into a dependency that's built against newlib. Take the opposite approach and try to use GLFW by afxgroup and it doesn't cleanly build unless I hack it about because it was build for clib2.

I need to start from a clean slate and get my head back into it. It isn't helping that I am still waiting on hardware but that's a different story.

**Comment by Guddler** [comment]
@walkero-gr Happy to report this is now incredibly fast. I've not been able to test the built library yet but that's nothing to do with the container at this point at least.
